### PR TITLE
Update code.org/donate with campaigns and supporters sections

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update https://code.org/donate with the following:
- Add the donor video in the top section
- Add a "Support a campaign" section
- Add a supporters section w/ company logos
- New sections are fully responsive and work with LTR and RTL languages
- I replaced all `jpg` logos with transparent background `png`s so they work on any background color
  - There were a few pages where these needed to be swapped so I did that too

**Jira ticket:** [ACQ-1064](https://codedotorg.atlassian.net/browse/ACQ-1064)

----

### Updates are getting pointed at by the red arrows
Artboard

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss [hunk 1]
```diff
@@ -674,6 +674,10 @@ form {
   display: flex;
   align-content: baseline;
 
+  &.white {
+    background: var(--neutral_white);
+  }
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss:679]: Added this class to use on `.action-block` elements that need a white background.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -1323,9 +1333,13 @@
   donate_page_top_desc: "Every donation fuels Code.org's mission to expand access to computer science education. We offer several ways for you to donate and make a difference."
   donate_page_fundraise_title: "Fundraise for Code.org"
   donate_page_fundraise_desc: "Help raise money for Code.org! Every student in every school should have the opportunity to learn computer science."
+  donate_page_campaigns_heading: "Support a campaign"
+  donate_page_campaigns_farsi_desc: "Talent is widespread, but opportunity often isn't. The proposed Code.org in Farsi initiative aims to grant over 110 million Farsi speakers, from countries like Iran, Afghanistan, and Tajikistan to global diasporas, access to educational resources currently available in other major languages. Let's make it a reality."
```
Comment (hannahbergam) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:1337]: My personal preference would be to remove the comma between 'speakers' and 'from', but that's just a nit!

File: pegasus/sites.v3/code.org/public/about/2021.md.partial [hunk 3]
```diff
@@ -298,14 +298,14 @@ Code.org gratefully acknowledges the following supporters who have made generous
 
 <h2>Platinum Supporters ($3,000,000+)</h2>
 <section class="flex-container">
-  <img src="/images/fit-150/avatars/amazon.jpg" alt="Amazon logo" />
```
Comment (hannahbergam) [inline pegasus/sites.v3/code.org/public/about/2021.md.partial:301]: I'm curious- did you upload the new ones as pngs so you wouldn't have to rename them?? If so, brilliant.

**Review by hannahbergam — APPROVED** [review]
LGTM! One nit, nonblocking

File: pegasus/sites.v3/code.org/public/about/2021.md.partial (hunk 3 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/about/2021.md.partial:301]: I did! 😄

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:1337]: Good call, I'll remove it 👍

**Pull request merged**